sys-libs/compiler-rt/compiler-rt-15.0.7-r2.ebuild

```
@@ -12,7 +12,8 @@ HOMEPAGE="https://llvm.org/"
 LICENSE="Apache-2.0-with-LLVM-exceptions || ( UoI-NCSA MIT )"
 SLOT="${LLVM_VERSION}"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86 ~amd64-linux ~ppc-macos ~x64-macos"
-IUSE="+abi_x86_32 abi_x86_64 +clang debug test"
+IUSE="+abi_x86_32 abi_x86_64 atomic-builtins +clang debug test"
```

```suggestion
IUSE="+abi_x86_32 abi_x86_64 +atomic-builtins +clang debug test"
```

Let's default it on?

---

Done